tests/desktop_chrome.pm

```diff
@@ -20,9 +20,11 @@ sub run {
     # long) and click the ‘Launch’ button from gnome-software to get to Chrome.
     if (check_screen('gnome_software_chrome_details_installed', 10)) {
         assert_and_click('gnome_software_chrome_details_installed', 'left', 10);
-        send_key('ret');  # skip through the ‘Make Google Chrome the default browser dialogue
     }
 
+    # Skip through the ‘Make Google Chrome the default browser’ dialogue.
+    send_key('ret');
```

Comment:
However, this doesn't seem to work. I think you may be racing between the dialog appearing and the test suite pressing ret?

Reply:
Fixed by adding a sleep and a comment.